public-api/src/main/java/org/pmiops/workbench/publicapi/GenomicsController.java

```diff
@@ -55,7 +59,41 @@
             "gvs_sas_ac as sas_allele_count, gvs_sas_an as sas_allele_number, gvs_sas_af as sas_allele_frequency, " +
             "gvs_oth_ac as oth_allele_count, gvs_oth_an as oth_allele_number, gvs_oth_af as oth_allele_frequency, " +
             "gvs_all_ac as total_allele_count, gvs_all_an as total_allele_number, gvs_all_af as total_allele_frequency from ${projectId}.${dataSetId}.wgs_variant";
-
+    private static final String FILTER_OPTION_SQL_TEMPLATE_GENE = "with a as\n" +
```

Comment:
This query fetches unique gene value, count
consequence value, count
clinical significance value, count
to display options on UI.